modules/local/kraken2preparation.nf

```diff
@@ -11,8 +11,8 @@ process KRAKEN2PREPARATION {
     tuple val(meta), path(db)
 
     output:
-    tuple val(meta), path( "database/" ), emit: db
-    path "versions.yml"                 , emit: versions
+    path( "database/" ) , emit: db
```

Comment:
```suggestion
    path "database/"  , emit: db
```

Not sure if you really need the brackets, and alignment might not fit anymore if you accept these changes!